### PR TITLE
fix: Minor grammar adjustments (no-changelog)

### PR DESCRIPTION
**Pull request by sandramsc**
Files edited: CONTRIBUTING.md, packages/node-dev/readme.md
 
   • Adjusted table of contents 
    • Added minor grammar and punctuation adjustments 
[merged sandramsc-patch-1-1 into sandramsc-patch-1 as per initial feedback]

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,7 +2,10 @@
 
 Great that you are here and you want to contribute to n8n
 
-## Contents
+## Table of Contents
+
+<details>
```
Comment (StarfallProjects) [inline CONTRIBUTING.md:7]: Let's not do this. Not much gained by hiding the TOC, and it forces another click for people who want to use it.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -85,7 +89,7 @@ With Node.js v16.17 or newer, you can install the latest version of pnpm: `corep
 
 #### Build tools
 
-The packages which n8n uses depend on a few build tools:
+The packages which n8n uses depend on a few build tools.
```
Comment (StarfallProjects) [inline CONTRIBUTING.md:92]:
```suggestion
The packages which n8n uses depend on a few build tools:
```

Keep the `:` - it's the start of a list of tools.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -171,18 +175,18 @@ automatically build your code, restart the backend and refresh the frontend
    ```
    pnpm dev
    ```
-1. Hack, hack, hack
-1. Check if everything still runs in production mode
+2. Hack, hack, hack
```
Comment (StarfallProjects) [inline CONTRIBUTING.md:178]: Suggest we keep the repeated `1.` as it makes it easier to edit the list with more/fewer steps in future. If anything, all the numbered lists should do this.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -226,4 +230,4 @@ That we do not have any potential problems later it is sadly necessary to sign a
 
 We used the most simple one that exists. It is from [Indie Open Source](https://indieopensource.com/forms/cla) which uses plain English and is literally only a few lines long.
 
-A bot will automatically comment on the pull request once it got opened asking for the agreement to be signed. Before it did not get signed it is sadly not possible to merge it in.
+Once a pull request is opened, an automated bot will promptly leave a comment requesting the agreement to be signed. Sadly, the pull request can only be merged once the signature is obtained.
```
Comment (StarfallProjects) [inline CONTRIBUTING.md:233]:
```suggestion
Once a pull request is opened, an automated bot will promptly leave a comment requesting the agreement to be signed. The pull request can only be merged once the signature is obtained.
```

**Review by StarfallProjects — CHANGES_REQUESTED** [review]
Thanks for the contribution! I've suggested a few edits.

**Comment by sandramsc** [comment]
@netroy Hi pardon the previous 3 closed PR's wasn't able to push local changes to remote and made them via GitHub UI instead. Thanks for the initial feedback.

File: CONTRIBUTING.md (hunk 3 above)
Comment (sandramsc) [reply CONTRIBUTING.md]: Not sure I understand or agree with this logic (perhaps something that is in practice and just haven't come across it till now), but I've gone ahead and reverted the change.

**Comment by sandramsc** [comment]
Requested changes have been implemented.

**Review by StarfallProjects — APPROVED** [review]

**Comment by StarfallProjects** [comment]
Thanks @sandramsc 🎉

**Comment by janober** [comment]
Got released with [`n8n@0.235.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%400.235.0)

**Pull request merged**